using-list-methods

```diff
@@ -0,0 +1,9 @@
+# In python we gt many options to combine elements of list into single string.
+
+arr = ['H','e','l','l','o',' ','W','o','r','l','d']   # initiating the list with elements
+
+print("".join(arr))  # using join method with double quote
+print(''.join(arr))  # using join method with single quote
```

Comment:
These methods have been already [implemented](https://github.com/Goku-kun/1000-ways-to-print-hello-world-in-python/blob/master/using-join.py). There is no difference between single/double quote which would be a redundant implementation in my opinion.